packages/labs/virtualizer/rollup.config.js

```diff
@@ -22,9 +22,7 @@ export default [
         warnings: true,
         module: true,
       }),
-      filesize({
-        showBrotliSize: true,
-      }),
+      summary(),
```

Comment:
Manually tested this and it looks great! The summary is missing the size of the compressed `brotli` values.

Worth adding back the `showBrotliSize` option?  https://yousifalraheem.github.io/rollup-plugin-summary/#:~:text=300000-,showBrotliSize,-%3F

Reply:
Actually, maybe we don't need this since it's not in the rollup-common config.

Reply:
It's true by default in `rollup-plugin-summary`

Reply:
Manually running `npm run checksize` only shows the size uncompressed, but I do see the `default true` documentation.

It looks like in the implementation the default is false (from a cursory look): https://github.com/yousifalraheem/rollup-plugin-summary/blob/main/src/index.ts#L20-L22

Reply:
Good catch, you're right! I added `showBrotliSize` and `showGzippedSize` to all the invocations.

Reply:
Filed https://github.com/yousifalraheem/rollup-plugin-summary/issues/52